### DISCUSSION TITLE
Signac- Stuart lab-Closestfeature-scATAC

### DISCUSSION THREAD

**Discussion by lprakas1** [stuart-lab/signac#1580] [Q&A] [answered]
Hello,

Thanks for providing such and amazing package for analysing multiomics data. I ahve used signac to identify overrepresented motifs within DA regions between two cell groups to identify a set of relevant transcription factors. Is there a way to go from the motif annotations to find genes which has these motifs enriched or in other way look for peaks for a given TF upstream or downstream to a given genomic region? I am aware of the closest feature function but dont know how to incorporate motif annotation in that function.
Alternatively, is there a way to find differentially accesible peak next to a given gene and map it to a transcription factor or its motif? 
 Thanks in advance.

**Comment by timoast** [comment, marked as answer]
Hi, this is a more general problem of finding overlapping or closest genomic regions. You should take a look at the GenomicRanges package, they have extensive support for doing these sorts of things: https://bioconductor.org/packages/release/bioc/html/GenomicRanges.html

You can also find the genomic coordinates of motif instances using the motifmatchr package: https://bioconductor.org/packages/release/bioc/html/motifmatchr.html
